python/desc/monitor/dbConnection.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from lsst.sims.catalogs.generation.db import DBObject
+
+class dbInterface(object):
+
+    def __init__(self):
```

Comment:
I thought that the `.lsst/db-auth.paf` mechanism, which we're using here, was sufficient, since that provides the user and password in a secure way.  Are you also worried about the host and port info?